### ISSUE TITLE
Namespaces not merged from UMD globals

### ISSUE THREAD

**Issue by RyanCavanaugh** [Microsoft/TypeScript#9329]
**a.d.ts**
```ts
export = ns;

export as namespace ns;

declare namespace ns {
  export var x: number;
  export interface IFoo { }
}
```

**b.d.ts**
```ts
declare namespace ns.something {
  export var p: ns.IFoo;
}
```

> `tsc a.d.ts b.d.ts`

Expected: No error
Actual: Error that `ns` has no member `IFoo` (because the `ns` UMD global gets shadowed rather than merged)

**Comment by RyanCavanaugh** [comment]
Talked with @mhegazy and we think this might be the behavior we want anyway -- `b.d.ts` should convert to UMD or module augmentation

**Comment by andy-ms** [comment]
It looks like this was done intentionally in #10354, which makes the `export as namespace` take no effect if there is already a global. @RyanCavanaugh @mhegazy Can this be closed?

**Comment by RyanCavanaugh** [comment]
This is the intended behavior, actually
